bucket/gatling.json

```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://gatling.io",
+    "description": "Async Scala-Akka-Netty based Load Test Tool",
+    "version": "3.3.1",
+    "license": "Apache-2.0",
+    "url": "https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/3.3.1/gatling-charts-highcharts-bundle-3.3.1-bundle.zip",
+    "hash": "sha1:6d9630ef1c3af22b217bb7cef3c6179924886469",
+    "extract_dir": "gatling-charts-highcharts-bundle-3.3.1",
+    "bin": [
+        "gatling.bat",
+        "recorder.bat"
+    ],
+    "persist": [
+        "user-files",
+        "results"
+    ],
+    "env_set": {
+        "GATLING_HOME": "$dir"
```

Comment:
Where is explicitly said you need this variable? Since scoop register binary, it should not be needed.

Reply:
When you run it: if you run recorder.bat without setting that environment variable, this happens:
![image](https://user-images.githubusercontent.com/3267561/69445532-6b782b80-0d4a-11ea-98e9-261c3c67586f.png)